app/assets/javascripts/discourse/app/lib/user-tips.js

```diff
@@ -25,6 +26,7 @@ export function showUserTip(options) {
     trigger: "manual",
     theme: "user-tips",
     zIndex: "",
+    delay: isTesting() ? 0 : 100,
```

Comment:
Just curious, any reason for the delay?

Reply:
No delay can cause the user tip to jump a bit while the page is rendered.

Reply:
Did you try wrapping this in a `schedule("afterRender", ...` block?

DOM operations should almost always happen inside this kind of block.

Reply:
Yes, I tried that. It was slightly better, but it still did not fix the issue.

Reply:
I added that now too because I find it to be a good suggestion nonetheless.